index.html

```diff
@@ -18,20 +18,21 @@
   <main>
     <header>
       <nav class="navbar">
-        <div class="navbar-content">
-          <a href="#" class="logo">My Logo</a>
-          <img src="pimages/menu button.png" />
-        </div>
-        <div class="nav_link hide">
-          <ul>
-            <li><a href="#">Portfolio</a></li>
-            <li><a href="#">About</a></li>
-            <li><a href="#">Contact</a></li>
+        <a href="#" class="logo">My Logo</a>
+          <ul class="nav-menu">
+            <li class="nav-content"><a href="#" class="nav-link">Hello</a></li>
+            <li class="nav-content"><a href="#sectina" class="nav-link">Portfolio</a></li>
+            <li class="nav-content"><a href="#aboutme" class="nav-link">About</a></li>
+            <li class="nav-content"><a href="#contactme" class="nav-link">Contact</a></li>
           </ul>
```

Comment:
- [ ] Kindly consider making sure that your  hamburger button and menu follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) to make your it an exact duplicate of the given design while doing so make sure the following look exactly the same as in the given design:
- hamburger color 
- nav links color
- padding and margins
check the images below for more clarification
- 1.your design 👇 
<img width="366" alt="Screen Shot 2023-05-22 at 18 06 36" src="https://github.com/Ejikeokegbe/Mobile-First-portifolio/assets/60146030/72f0f6ad-bf66-4022-9162-65f1415283ab">

- 2.Figma design 👇 
<img width="342" alt="Screen Shot 2023-05-22 at 18 07 31" src="https://github.com/Ejikeokegbe/Mobile-First-portifolio/assets/60146030/47f87845-81d6-4b3a-b53f-979637a944ab">

Reply:
- [ ] As suggested earlier, kindly ensure that your design is close to the given template. 👍 

- [ ] Kindly take note of the following:

1. Appropriate margin (left, right and top) arrond the nav items. ℹ️ 
2. Implement the background with CSS blur properties. ℹ️ 
3. Spacing between the Portfolio and the close (X) button. ℹ️ 

## Your Design

<img width="327" alt="Untitled2" src="https://github.com/Ejikeokegbe/Mobile-First-portifolio/assets/103386183/8b980f24-2695-42c0-be8f-eaba068f6ed8">

## Figma Design

<img width="141" alt="Untitled2" src="https://github.com/Ejikeokegbe/Mobile-First-portifolio/assets/103386183/2b4aaad8-1db9-44dc-937e-74a84dbbfe86">